app/src/androidTest/java/org/sil/gatherwords/MainActivityTest.java

```diff
@@ -0,0 +1,43 @@
+package org.sil.gatherwords;
+
+import android.content.Context;
+import android.os.Looper;
+import android.support.test.InstrumentationRegistry;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.sil.gatherwords.room.AppDatabase;
+import org.sil.gatherwords.room.SessionDao;
+import org.sil.gatherwords.room.WordDao;
+
+/**
+ * Test the Main Activity
+ */
+public class MainActivityTest{
```

Comment:
What is the purpose of this test class; it contains no tests?

Reply:
Sorry, I missed removing this from the index